modin/pandas/series.py

```diff
@@ -2107,9 +2107,13 @@ def unique(self) -> ArrayLike:  # noqa: RT01, D200
         """
         Return unique values of Series object.
         """
-        return self.__constructor__(
-            query_compiler=self._query_compiler.unique()
-        ).to_numpy()
+        # `values` can't be used here because it performs unnecessary conversion,
+        # after which the result type does not match the pandas
+        return (
+            self.__constructor__(query_compiler=self._query_compiler.unique())
+            .modin.to_pandas()
+            ._values
```

Comment:
Is there a public method that returns the same?

Reply:
I searched but didn't find it (`values` and `array` are not suitable).